domain-models-runtime/src/main/java/org/folio/rest/persist/Criteria/Criteria.java

```diff
@@ -235,164 +141,73 @@ private String addPrefix() {
       return prefix + GET_JSON_FIELD;
     } else {
       // top level field criteria
-      if(isArray){
+      if (isArray) {
         return "";
-      }
-      else{
+      } else {
         return prefix + GET_VALUE_FIELD;
       }
     }
   }
 
   /**
-   * Added fields in arrayList to a string - prefix and -> / ->> added in addPrefix()
+   * Added fields in arrayList to a string - prefix and -> / ->> added in
+   * addPrefix()
+   *
    * @return
    */
-  private String field2String(boolean validate){
+  private String field2String() {
     StringBuilder sb = new StringBuilder();
     int size = field.size();
-    if(size == 1 && isArray){
+    if (size == 1 && isArray) {
       //query where claue should look like -
       return from.asValue;
     }
-    if(size == 1){
+    if (size == 1) {
       return field.get(0);
     }
     for (int i = 0; i < size; i++) {
-      if(isArray){
+      if (isArray) {
         //skip first field if array - included in FROM clause with an AS
-        if(i==0){
+        if (i == 0) {
           i++;
         }
       }
       String fVal = field.get(i);
```

Comment:
Can you remove `fVal` and use `field.get(i)` directly?

Reply:
Sure